src/components/a-checkbox/index.js

```diff
@@ -145,4 +145,6 @@ class Checkbox extends BaseComponentGlobal {
   }
 }
 
-window.customElements.define('axa-checkbox', Checkbox);
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
I think it's time for a `ready` function

Reply:
Will do